lib/pipeline/addon/pipelines/project/controller.js

```diff
@@ -0,0 +1,111 @@
+import { scheduleOnce } from '@ember/runloop';
```

Comment:
Mount into existing project or re-export the app controller... this is all duplicate and most of it not relevant to pipelines.